docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -15,7 +15,7 @@ weight = 3
 
  - The Remote API has replaced `rcli`.
  - The daemon listens on `unix:///var/run/docker.sock` but you can
-   [Bind Docker to another host/port or a Unix socket](../commandline/dockerd.md#bind-docker-to-another-host-port-or-a-unix-socket).
+   [Bind Docker to another host/port or a Unix socket](../commandline/dockerd.md#bind-docker-to-another-hostport-or-a-unix-socket).
```

Comment:
Hm, I guess you tried this link on GitHub, and not on the https://docs.docker.com website? Unfortunately there's some minor differences between GitHub and the software we use to generate our online docs. The old link is working when viewed online; https://docs.docker.com/engine/reference/commandline/dockerd/#bind-docker-to-another-host-port-or-a-unix-socket

Since the primary purpose of these files is to generate our online docs (and not to be read on GitHub), could you change these back?

Also /cc @SvenDowideit as we were looking into this through https://github.com/docker/docs-base/issues/186